--- wscript	2010-04-24 08:14:31.000000000 +0530
+++ wscript.new	2010-04-24 08:14:22.000000000 +0530
@@ -136,7 +136,7 @@
                        uselib_store='GNOME_DESKTOP', mandatory=True,
                        args='--cflags --libs')
     # make sure we have the proper Vala version
-    if conf.env['VALAC_VERSION'] != MIN_VALA_VERSION and \
+    if conf.env['VALAC_VERSION'] < MIN_VALA_VERSION and \
         not os.path.isdir(os.path.join(conf.curdir, GEN_SRC_DIR)):
         conf.fatal('''\
 Your Vala compiler version %s is too old. The project requires